tsrelay/handler/get_serve.go

```diff
@@ -144,14 +144,13 @@ func (h *handler) getServe(ctx context.Context, body io.Reader) (*serveStatus, e
 			TailscaleIPs: st.Self.TailscaleIPs,
 		}
 
-		capabilities := st.Self.Capabilities
-		if slices.Contains(capabilities, tailcfg.CapabilityWarnFunnelNoInvite) ||
-			!slices.Contains(capabilities, tailcfg.NodeAttrFunnel) {
+		if st.Self.HasCap(tailcfg.CapabilityWarnFunnelNoInvite) ||
+			!st.Self.HasCap(tailcfg.NodeAttrFunnel) {
```

Comment:
I don't think this will work for people who have not upgraded their `tailscaled` -- we might need to check _both_ the old field and the new field. I think you're already doing this for the URL port parsing below, so this should be consistent. Might be worth abstracting away, or maybe even checking the version of the local backend and act accordingly?

Reply:
`HasCap` does check both the old and new fields -- see https://github.com/tailscale/tailscale/blob/ca4396107eb650e40868e2d4a68bc13b0d79c123/ipn/ipnstate/ipnstate.go#L298. The reason I don't use `HasCap` below is that it doesn't check for prefixes, only equality.

Reply:
fwiw I tested this with 1.48.2, and @tylersmalley tested this with some older clients as well, with success.